logic/WebRequesters/SpigotWebRequester.cs

```diff
@@ -62,8 +93,8 @@ public async Task<List<ServerVersion>> RequestSpigotVersions()
             {
                 ServerVersion serverVersion = new ServerVersion();
                 serverVersion.Type = ServerVersion.VersionType.Spigot;
-                serverVersion.Version = version.version;
-                serverVersion.JarLink = "https://serverjars.com/api/fetchJar/spigot/" + version.version;
+                serverVersion.Version = version;
+                serverVersion.JarLink = $"https://cdn.getbukkit.org/spigot/spigot-{version}-R0.1-SNAPSHOT-latest.jar";
```

Comment:
I am not familiar with the API of getbukkit, but "-SNAPSHOT-" indicates a snapshot build which means, that you allways download the latest snapshot build of the version, but not the actual stable version right?

Reply:
I honestly don't know, I'll check rn